components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -2157,7 +2157,10 @@ spec:
             namespace }}/pipelinerun/{{ pr }}/logs/{{ task }}
   profile: all
   pruner:
-    disabled: true
+    keep: 10
```

Comment:
just realized https://github.com/redhat-appstudio/infra-deployments/pull/4184/files only affects the prod overlay @enarha  .... I'll defer to you on what you want to do with this older version of the pruner config that we used to use 6 or so months ago.

Reply:
went ahead and pulled in the keep-since related changes with https://github.com/openshift-pipelines/pipeline-service/pull/1031/commits/dfe7fc945780546d87166a7f2a5c2c7105a8bc6c